### PR TITLE
Producten filteren

### PR DESCRIPTION
**Pull request by samirelma**
-categories on navbar fixed
-filter by price added

### CODE REVIEW

File: src/components/nav.php [hunk 1]
```diff
@@ -199,8 +199,10 @@
         $categories = fetch('SELECT * FROM product_categories LIMIT 10');
         if ($categories) {
           foreach ($categories as $category) {
+            
             echo '
-            <a href="/catalog/products?category=' . $category['name'] . '" class="group flex flex-col gap-2 items-center">
+            
+            <a href="/catalog/products?category=' . $category['id'] . '" class="group flex flex-col gap-2 items-center">
```
Comment (CedricVerlinden) [inline src/components/nav.php:205]: Use the category name instead of the id

File: src/public/catalog/products.php [hunk 2]
```diff
@@ -4,19 +4,84 @@
...
+      </div> 
+      <br>
+      <div class="col-md-4"> 
+        <button class="btn btn-primary">Click Me</button>
+      </div> 
+    </div>
+    <input type="hidden" name="search" value="';
+    if(isset($_GET['search'])) echo $_GET['search'];
+    echo '">
+  </form>
```
Comment (CedricVerlinden) [inline src/public/catalog/products.php:67]: Use DaisyUI components

**Review by CedricVerlinden — CHANGES_REQUESTED** [review]

File: src/public/catalog/products.php (hunk 2 above)
Comment (samirelma) [reply src/public/catalog/products.php]: done

File: src/components/nav.php (hunk 1 above)
Comment (samirelma) [reply src/components/nav.php:205]: does not work when i use category name because it doesn't show any product then.

**Review by CedricVerlinden — APPROVED** [review]
LGTM! Great job

**Pull request merged**